pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1414,6 +1414,42 @@ self: super: {
   # Testsuite trying to run `which haskeline-examples-Test`
   haskeline_0_8_0_0 = dontCheck super.haskeline_0_8_0_0;
 
+  insert-ordered-containers = super.insert-ordered-containers.override {
+    optics-core = self.optics-core_0_3_0_1;
+    optics-extra = self.optics-extra_0_3;
+  };
+
+  # Tests depend on HTF and it is broken, 2020-08-23
+  list-t = dontCheck super.list-t;
+  superbuffer = dontCheck (doJailbreak super.superbuffer);
```

Comment:
Do you have a comment as to why wee need a jailbreak here?

Reply:
All of them have the same root cause. I just updated it to have the same comment each time so it is not lost on a refactor.

Reply:
Well, sorry, for nitpicking. You don‘t need to repeat the comment, I think. `dontCheck` is not a huge sin. I am more worried about the `doJailbreak` and I don‘t understand how your comment addresses that. Can you make that a bit more explicit?

Reply:
I removed the doJailbreak and it compiled, taking it out of the PR